src/framing/mod.rs

```diff
@@ -244,7 +245,7 @@ impl MLSCiphertext {
     pub(crate) fn to_plaintext(
         &self,
         ciphersuite: &Ciphersuite,
-        roster: &[&Credential],
+        indexed_members: HashMap<LeafIndex, &Credential>,
```

Comment:
Is there a particular reason we using a `HashMap` here as opposed to a slice?

Reply:
We could use a slice as well, but then we would have to account for blank leaves and work with options. A `HashMap` seemed the better fit here.

Reply:
I see. Good point!